packages/full-stack-tests/src/unified-selection/SelectionScope.test.ts

```diff
@@ -22,7 +22,9 @@ import { ECSchemaRpcInterface } from "@itwin/ecschema-rpcinterface-common";
 import { ECSchemaRpcImpl } from "@itwin/ecschema-rpcinterface-impl";
 import { createECSqlQueryExecutor } from "@itwin/presentation-core-interop";
 import { buildTestIModel, initialize, terminate } from "@itwin/presentation-testing";
-import { SelectableInstanceKey, SelectionScope } from "@itwin/unified-selection";
+import { computeSelection, SelectableInstanceKey } from "@itwin/unified-selection";
+
+type SelectionScope = "element" | "category" | "model" | "functional";
```

Comment:
It's okay for tests to import `@internal` API from our own packages.

Reply:
Not sure why I did not import it, considering I've done that for other tests already.